<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoreAddWidget</class>
 <widget class="QWidget" name="StoreAddWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>26</y>
     <width>212</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="StoreName_label">
      <property name="text">
       <string>店  名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="StoreName_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>58</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="StoreLeader_label">
      <property name="text">
       <string>负责人：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="StoreLeader_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>122</y>
     <width>212</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="StoreTel_label">
      <property name="text">
       <string>电  话：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="StoreTel_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>212</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="StoreAddress_label">
      <property name="text">
       <string>地  址：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="StoreAddress_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>187</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="Confirm_pushButton">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Cancel_pushButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
